.travis.yml

```diff
@@ -45,6 +46,7 @@ before_script:
 
 # command to run tests, e.g. python setup.py test
 script:
+  - jslint **/*.js
```

Comment:
I think specifying the directory would make it less intimidating:
https://github.com/openwisp/django-freeradius/tree/master/django_freeradius/static/django-freeradius

Reply:
I have updated it as per request.